app/services/foods_service.rb

```diff
@@ -0,0 +1,24 @@
+class FoodsService 
+  class << self
+
+    def foods_search(food)
+      response = conn.get('/fdc/v1/foods/search') do |req|
+        req.params['query'] = "ingredients:#{food}"
+        req.params['pageSize'] = 10
+      end
```

Comment:
I like how you're adding the pageSize into your query here since, for now, we know we only want to display 10 foods from this search.